scripts/cypress.js

```diff
@@ -1,82 +0,0 @@
-#!/usr/bin/env node
```

Comment:
I dont have much context with cypress stuff so curious as to why was this file removed?

Reply:
I talked with @ling1726 and its a leftover, this is not used at all - that's why removing